.snyk

```diff
@@ -6,5 +6,9 @@ ignore:
     - '*':
         reason: No replacement available
         expires: 2022-07-31T00:00:00.000Z
+  SNYK-JAVA-IONETTY-2812456:
```

Comment:
This does have a fix doesn't it? Did we already fix it in grpc utils ? I know we fixed some netty deps on 0.7.3 but on phone and hard to check

Reply:
This seems to be coming from different paths, not just GRPC utils. So, sync. fails without this exclusion.

Reply:
```
2022-05-23T08:50:53.9177667Z Issues with no direct upgrade or patch:
2022-05-23T08:50:53.9178296Z   ✗ Information Exposure [Medium Severity][https://snyk.io/vuln/SNYK-JAVA-IONETTY-2812456] in io.netty:netty-common@4.1.72.Final
2022-05-23T08:50:53.9179196Z     introduced by io.grpc:grpc-netty@1.45.1 > io.netty:netty-codec-http2@4.1.72.Final > io.netty:netty-common@4.1.72.Final and 82 other path(s)
2022-05-23T08:50:53.9180039Z   This issue was fixed in versions: 4.1.77.Final
```
Should we add a constraint instead?

Reply:
No, grpc utils upgrade handles it